app/javascript/components/reconfigure-vm-form/network-table/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MiqDataTable from '../../miq-data-table';
+
+const NetworkTable = ({
+  headers, rows, onNetworkCellClick, buttonClick,
+}) => (
+  <div>
+    <hr />
+    <h3>{__('Network Adapters')}</h3>
```

Comment:
I think its better to say `Add Network Adapters` or `Edit Network Adapters`

Reply:
Made the form title changes.